appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
@mark-friedman I was wondering if you had gotten a chance to look at this code/think about it. My progress is getting severely slowed down, and I'm not quite sure what to do. Also, if you have any tips on how I can debug this code, that would really be useful because right now I know of no way of doing that other than building an app and seeing it works.